stock_picking_product_generate_sequence/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+##############################################################################
+# Copyright (c) 2022 brain-tec AG (https://braintec.com)
+# All Right Reserved
+#
+# See LICENSE file for full licensing details.
+##############################################################################
+{
+    'name': 'Stock Picking Product Generate sequence',
+    'version': '15.0.1.0.0',
+    'summary': 'Add an automated sequence in product of stock picking',
+    'category': 'Inventory',
+    'author': 'brain-tec AG',
+    'website': 'https://braintec.com',
+    'depends': [
+        'product',
+        'stock',
+    ],
+    'license': 'OPL-1',
```

Comment:
[Major] If we will contribute with it to the OCA I'm afraid OPL-1 is not a good choice. If we won't contribute then better to place it in the project repo